crates/apub/src/objects/comment.rs

```diff
@@ -2,23 +2,20 @@ use crate::{
   activities::{verify_is_public, verify_person_in_community},
   check_apub_id_valid_with_strictness,
   fetch_local_site_data,
-  local_instance,
   mentions::collect_non_local_mentions,
   objects::{read_from_string_or_source, verify_is_remote_object},
   protocol::{
     objects::{note::Note, LanguageTag},
     InCommunity,
     Source,
   },
-  PostOrComment,
 };
 use activitypub_federation::{
-  core::object_id::ObjectId,
-  deser::values::MediaTypeMarkdownOrHtml,
-  traits::ApubObject,
-  utils::verify_domains_match,
+  config::Data,
+  kinds::{object::NoteType, public},
+  protocol::{values::MediaTypeMarkdownOrHtml, verification::verify_domains_match},
+  traits::Object,
```

Comment:
This is in your crate, but I def recommend using another name, to be more specific about what type of Object / the functions this trait does.

Reply:
I found the "Apub" prefixes really ugly and removed them. Anyway there are no name conflicts in the files that use them, and the meaning is clear from the context.